packages/core/src/constants.ts

```diff
@@ -13,7 +13,7 @@ export const IS_PRODUCTION = process.env.NODE_ENV === "production"
 
 export const DEFAULT_SERVICE_ENDPOINT: HTTPSURLString = IS_PRODUCTION
   ? "https://ads.mozilla.org/" // production
-  : "https://ads.allizom.org/" // staging
+  : "https://ads.mozilla.org/" // staging
```

Comment:
ehhh.. can we just update playwright to set `NODE_ENV=production` instead?

Reply:
Yes, apologies for the mess, that would absolutely be the right way to do it, I will make that change. This was just me trying out a quick way to make sure this approach would work